samples/videoDecode/videodecode.cpp

```diff
@@ -57,7 +58,8 @@ void ShowHelpAndExit(const char *option = NULL) {
     << "-seek_criteria - Demux seek criteria & value - optional; default - 0,0; "
     << "[0: no seek; 1: SEEK_CRITERIA_FRAME_NUM, frame number; 2: SEEK_CRITERIA_TIME_STAMP, frame number (time calculated internally)]" << std::endl
     << "-seek_mode - Seek to previous key frame or exact - optional; default - 0"
```

Comment:
I think, it is better to add another sample called "video_es_decode" for this instead of clubbing it with videodecode sample

Reply:
Agreed. The current code in videodecode.cpp is just a demo sample. When we remove ffmpeg links, we will need a new sample.